composer.json

```
@@ -7,6 +7,6 @@
 		"predis/predis": "~1.1"
 	},
 	"require-dev": {
-		"phpunit/phpunit": "4.5.0"
+		"phpunit/phpunit": "7.0.0"
```

We can use the constraint `^7.1.4` here instead:
```suggestion
		"phpunit/phpunit": "^7.1.4"
```